celery_utils/management/commands/drop_djcelery_tables.py

```diff
@@ -0,0 +1,80 @@
+"""
+This command will drop the tables included in django-celery's
+migrations, which are included when we bump that to 3.2.1
+
+If the tables are not empty, this command will show a message and exit.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import OrderedDict
+import logging
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from djcelery.models import (CrontabSchedule, IntervalSchedule, PeriodicTask,  # pylint:disable=import-error
```

Comment:
I have, and or CI tests for its correctness, but somehow (on Travis) the commit and the merge commit differ in whether this is correct or not?

https://travis-ci.org/edx/edx-celeryutils/jobs/238842076
https://travis-ci.org/edx/edx-celeryutils/jobs/238842067